### DISCUSSION TITLE
Reactivate KepassXC_Browser entry; accidentally deactivated and noted

### DISCUSSION THREAD

**Discussion by Rafaelito2609** [keepassxreboot/keepassxc#11284] [Q&A]
I had saved several login accounts on a website (in the connected KeepassXC database on my PC in myPasswords.kdbx).
because every time I logged in to this website, so many were offered by KeepassXC_Browser extension, I clicked on various things to stop displaying them, deactivate them and remember them..
Unfortunately, I accidentally deactivated the right and last one. Now I can't find a way to display it again in the browser?
-> the login data exists in the database
-> I completely deleted the browser cache (all cookies..)
-> database disconnected and reconnected

but even after retyping the login data several times by hand, KeepassXC Browser_Extension does not recognize the data

how can I activate it again?
the error message always appears:

Error! Error: No applications found. htps://www.beispiel.com

**Comment by droidmonkey** [comment]
![image](https://github.com/user-attachments/assets/686b0547-ebfc-4058-8192-41ddfa72e50f)

**Comment by Rafaelito2609** [comment]
THANKS
i found it now in the propertys of "KeepassXC" on my desktop computer.
thank you very much !!
